### PR TITLE
Some 6502 target changes

### PR DESCRIPTION
**Pull request by groessler**
- platform/atarixl/Makefile.atarixl: put two more objects into high memory
- platform/atarixl/contiki-conf.h: define WWW_CONF_WGET_EXEC
- tools/6502/Makefile: add 'clean' target; remove unused CONTIKI define

### CODE REVIEW

File: platform/atarixl/Makefile.atarixl [hunk 1]
```diff
@@ -35,7 +35,7 @@ CONTIKI_CPU = $(CONTIKI)/cpu/6502
 include $(CONTIKI_CPU)/Makefile.6502
 
 SHADOW_RAM_SOURCEFILES  = ethernet.c ethernet-drv.c timer.c uip.c uiplib.c
-SHADOW_RAM2_SOURCEFILES = clock.c uip_arch.c uip_arp.c
+SHADOW_RAM2_SOURCEFILES = clock.c uip_arch.c uip_arp.c unload.c random.c
```
Comment (greg-king5) [inline platform/atarixl/Makefile.atarixl]: Please look at https://github.com/contiki-os/contiki/pull/1265/files#diff-a59eec8547717a09a0ad24ff500a7600

I needed to move some objects, in order to make PR #1265 fit into the apple2enh and atarixl platforms.  I hope that you will include those moves here (or, make an even better fitting map for that PR).

File: tools/6502/Makefile [hunk 2]
```diff
@@ -48,6 +44,7 @@ ifndef DIR2ATR
 endif
 
 all: apple2 atari c64 c128
+clean: apple2-clean atari-clean c64-clean c128-clean
```
Comment (greg-king5) [inline tools/6502/Makefile]: It would be nice to have `.PHONY` targets for the make-file's non-file goals.

Comment (groessler) [inline tools/6502/Makefile]: Ok, done.

File: platform/atarixl/Makefile.atarixl (hunk 1 above)
Comment (groessler) [inline platform/atarixl/Makefile.atarixl]: I'd like to keep this out ouf this PR. I'll look at it in a separate PR.

**Comment by oliverschmidt** [comment]
The failing Travis CI jobs are not related to the changes in this pull request.

**Pull request merged**